### ISSUE TITLE
Can you offer the pre-trained model weights?

### ISSUE THREAD

**Issue by liupgd** [mousecpn/Joint-Bilateral-Learning#7]
Hi, I'm interested with this model. But I can't run the test without your pretrained weights. Appreciated if you can offer the pretrained weights. Thx.
My email: liupgd@foxmail.com

**Comment by mousecpn** [comment]
> Hi, I'm interested with this model. But I can't run the test without your pretrained weights. Appreciated if you can offer the pretrained weights. Thx.
> My email: [liupgd@foxmail.com](mailto:liupgd@foxmail.com)

Sorry, I didn't train successfully on COCO dataset.

**Comment by liupgd** [comment]
> > Hi, I'm interested with this model. But I can't run the test without your pretrained weights. Appreciated if you can offer the pretrained weights. Thx.
> > My email: [liupgd@foxmail.com](mailto:liupgd@foxmail.com)
> 
> Sorry, I didn't train successfully on COCO dataset.

 I trained on COCO2017, but the result is far away from the paper. I used 100k images, same data operation with the paper. I checked your code and the BGU paper and the JBL paper. I think there is almost no fault with your code.   
Here is my result trained 29 epochs:

![individualImage](https://user-images.githubusercontent.com/3350539/97651040-0d91af00-1a96-11eb-8d9d-0dc470ef6be4.png)

All the test images are searched with google. The first two results are not as good as the paper. I don't know why. Maybe, the training dataset also matters.

**Comment by oferidan1** [comment]
hi liupgd, mousecpn , can you please upload your pre-trained model?
I understand its not perfect, but it would be great if you can share it.
thanks in advanced.

**Comment by liupgd** [comment]
> hi liupgd, mousecpn , can you please upload your pre-trained model?
> I understand its not perfect, but it would be great if you can share it.
> thanks in advanced.

Hi @oferidan1 , I modified the training to pytorch-lightning. I've merged my code to here: https://github.com/liupgd/Joint-Bilateral-Learning
checkpoint: https://drive.google.com/file/d/1BH5RyvRcwm0VC75iEcp2NLe4V7imyAn2/view?usp=sharing
some parameters: 
![image](https://user-images.githubusercontent.com/3350539/116534971-c89be780-a915-11eb-84a5-2b988d90fc08.png)

It has been so long, I can't remember all details now. All above is what I can find out from my computer. Hope it will help you.

**Comment by oferidan1** [comment]
thanks alot, liupgd.

**Comment by lblbk** [comment]
> > hi liupgd, mousecpn , can you please upload your pre-trained model?
> > I understand its not perfect, but it would be great if you can share it.
> > thanks in advanced.
> 
> Hi @oferidan1 , I modified the training to pytorch-lightning. I've merged my code to here: https://github.com/liupgd/Joint-Bilateral-Learning
> checkpoint: https://drive.google.com/file/d/1BH5RyvRcwm0VC75iEcp2NLe4V7imyAn2/view?usp=sharing
> some parameters:
> ![image](https://user-images.githubusercontent.com/3350539/116534971-c89be780-a915-11eb-84a5-2b988d90fc08.png)
> 
> It has been so long, I can't remember all details now. All above is what I can find out from my computer. Hope it will help you.

你好 能讲中文吧 
我想问一下数据集是怎么做的 是随机在coco2017中选取风格图片和内容图片各50000张吗

**Comment by liupgd** [comment]
好像是的，好久前的事了，记得不太清了。-------- 原始邮件 --------发件人： martin_chen ***@***.***>日期： 2021年6月17日周四 19:43收件人： mousecpn/Joint-Bilateral-Learning ***@***.***>抄送： liupgd ***@***.***>, Author ***@***.***>主    题： Re: [mousecpn/Joint-Bilateral-Learning] Can you offer the pre-trained model weights? (#7)





hi liupgd, mousecpn , can you please upload your pre-trained model?

I understand its not perfect, but it would be great if you can share it.

thanks in advanced.



Hi @oferidan1 , I modified the training to pytorch-lightning. I've merged my code to here: https://github.com/liupgd/Joint-Bilateral-Learning

checkpoint: https://drive.google.com/file/d/1BH5RyvRcwm0VC75iEcp2NLe4V7imyAn2/view?usp=sharing

some parameters:



It has been so long, I can't remember all details now. All above is what I can find out from my computer. Hope it will help you.



你好 能讲中文吧

我想问一下数据集是怎么做的 是随机在coco2017中选取风格图片和内容图片各50000张吗

**Comment by zhanghongyong123456** [comment]
> > hi liupgd, mousecpn , can you please upload your pre-trained model?
> > I understand its not perfect, but it would be great if you can share it.
> > thanks in advanced.
> 
> Hi @oferidan1 , I modified the training to pytorch-lightning. I've merged my code to here: https://github.com/liupgd/Joint-Bilateral-Learning checkpoint: https://drive.google.com/file/d/1BH5RyvRcwm0VC75iEcp2NLe4V7imyAn2/view?usp=sharing some parameters: ![image](https://user-images.githubusercontent.com/3350539/116534971-c89be780-a915-11eb-84a5-2b988d90fc08.png)
> 
> It has been so long, I can't remember all details now. All above is what I can find out from my computer. Hope it will help you.
 
I use yours pre model ,but get error,i dont know why ,can  you  help me  ?
my   test cmd   

python test.py --cont_img_path /home/zhy/Projects/RealHM/vendor_testing_2/pexels-aleksandar-pasaric-2115367.jpg --style_img_path /home/zhy/Projects/RealHM/vendor_testing_2/pexels-aleksandar-pasaric-2115367_fore.jpg  --model_checkpoint ./checkpoints/JBL-epoch=73.ckpt

and error:

![image](https://user-images.githubusercontent.com/48466610/151287669-c4d42aad-5587-4ed0-bc1c-04f7eb9f699f.png)

**Comment by liupgd** [comment]
Hi,@zhanghongyong123456 if you are using both  my pytorch-lightning version of code and the pretrained model. please just use main_pl.py for training and testing purpose. The pytorch-lightning models are different from the original ones and can't be loaded by pytorch directly. The main_pl.py' s arguments has shown how to do the test. Hope this help you.

**Comment by zhanghongyong123456** [comment]
> Hi,@zhanghongyong123456 if you are using both my pytorch-lightning version of code and the pretrained model. please just use main_pl.py for training and testing purpose. The pytorch-lightning models are different from the original ones and can't be loaded by pytorch directly. The main_pl.py' s arguments has shown how to do the test. Hope this help you.

yes,I want to know how the parameters are configured, I only have one GPU, I don't know where to pass the test image (content and style)，only add checkpoint  path 

![image](https://user-images.githubusercontent.com/48466610/151500672-03ed8330-4c1d-4855-90f6-93a3d940497e.png)

**Comment by zhanghongyong123456** [comment]
> Hi,@zhanghongyong123456 if you are using both my pytorch-lightning version of code and the pretrained model. please just use main_pl.py for training and testing purpose. The pytorch-lightning models are different from the original ones and can't be loaded by pytorch directly. The main_pl.py' s arguments has shown how to do the test. Hope this help you.

Hello, I have successfully tested the model according to your prompt, I have two questions looking forward to your reply, 1. Can I achieve 4K real-time 1080 on the computer（in paper,I know   mobile phone  4k real time ?） 2. How to set the high resolution (2K 4K), the result I get is 512 Resolution, if it can be any resolution, set it there, I am not too familiar with the pytorch_lightning library
